### PR TITLE
HIVE-27704 : Remove PowerMock from jdbc-handler and upgrade mockito to 4.11

### PR DESCRIPTION
**Pull request by rkirtir**
### What changes were proposed in this pull request?
(https://issues.apache.org/jira/browse/HIVE-27704)

### Why are the changes needed?
To upgrade mockito

### Does this PR introduce _any_ user-facing change?
NO

### Is the change a dependency upgrade?
YES

### How was this patch tested?
Precommit tests

### CODE REVIEW

**Review by InvisibleProgrammer — APPROVED** [review]
LGTM

File: jdbc-handler/src/test/java/org/apache/hive/storage/jdbc/TestJdbcInputFormat.java [hunk 1]
```diff
@@ -45,20 +45,33 @@
...
 
   @Mock
   private DatabaseAccessor mockDatabaseAccessor;
+  private MockedStatic<DatabaseAccessorFactory> hiveDatabaseAccessorFactory;
+
+  @Before
+  public void setup() throws Exception{
+
+    hiveDatabaseAccessorFactory = mockStatic(DatabaseAccessorFactory.class);
+    hiveDatabaseAccessorFactory.when(() -> DatabaseAccessorFactory.getAccessor(any(Configuration.class))).thenReturn(mockDatabaseAccessor);
```
Comment (ayushtkn) [inline jdbc-handler/src/test/java/org/apache/hive/storage/jdbc/TestJdbcInputFormat.java:62]: line length allowed is 120 only
https://github.com/apache/hive/blob/master/checkstyle/checkstyle.xml#L160

**Review by ayushtkn — APPROVED** [review]
code isn't formatted properly, please format the code properly,
 rest changes LGTM

File: jdbc-handler/src/test/java/org/apache/hive/storage/jdbc/TestJdbcInputFormat.java (hunk 1 above)
Comment (rkirtir) [reply jdbc-handler/src/test/java/org/apache/hive/storage/jdbc/TestJdbcInputFormat.java]: fixed

**Pull request merged**